dts/bindings/adc/ti,ads114s08.yaml

```diff
@@ -26,5 +26,21 @@ properties:
     description: |
       GPIO for start/sync, used to signal the ADC that a conversion should be started
 
+  idac-current:
+    type: int
+    enum:
+      - 0
+      - 10
+      - 50
+      - 100
+      - 250
+      - 500
+      - 750
+      - 1000
+      - 1500
+      - 2000
+    default: 0
+    description: IDAC current in microampere
```

Comment:
The default value needs a justification in the description

Reply:
done (hopefully sufficiently detailed)